### ISSUE TITLE
[Easy Import] Cannot change configuration (LINUX only)

### ISSUE THREAD

**Issue by FJBDev** [mytourbook/mytourbook#457]
When trying to switch from an import configuration to another, nothing happens.

After several hours of debugging/investigation, I found the below error happens ONLY in Linux on this line of code in [RawDataView](https://github.com/mytourbook/mytourbook/blob/61d4f45d1ba5bcabe85db1013be7c53833ebc5d7/bundles/net.tourbook/src/net/tourbook/ui/views/rawData/RawDataView.java#L656)

```
SWT WebKitGDBus: there was an error executing something synchronously with the extension.
SWT WebKitGDBus: the error message is: The connection is closed
SWT webkit: failure registering BrowserFunction onSelectImportConfig
```

MyTourbook Version: **22.1**

@wolfgang-ch @telemaxx  Can you reproduce the issue on Linux ?

**Comment by FJBDev** [comment]
My next step is to find in which version it stopped working because I am pretty sure it was working before. My hunch is that it's related to the Eclipse RCP version.

**Comment by wolfgang-ch** [comment]
I just tested MT 22.1 with Ubuntu 20.04 and it works. I think that the change of the Webkit version in your Linux is causing this issue.

**Comment by wolfgang-ch** [comment]
Search for `SWT webkit: failure registering BrowserFunction` and you will find other people having this issue

**Comment by FJBDev** [comment]
Indeed, it seems to be an issue with Webkit

I found a partial solution [here](https://stackoverflow.com/questions/67574597/swt-browserfunction-not-registering/67591313#comment125350904_67591313) but per my testing, while it fixes the Easy Import issue, it creates other issues in my Debian so I decided to go back to the actual version Hopefully this issue will be fixed in the next versions of WebKit.
At least, if another person has this issue, then we now know what it the root cause.

**Comment by FJBDev** [comment]
@wolfgang-ch Can you show me which file I can manually modify to start MT with a specific Easy Import Config ? Is it in `workbench.xmi` ? 
That way, I can still use my different Easy Import configs. I will need to restart MT and modify the file when I want to change the config but at least that should work.

**Comment by wolfgang-ch** [comment]
It's not easy to modify the easy import config manually, you can find it

* In the file `C:\Users\<YourName>\mytourbook\.metadata\.plugins\net.tourbook\dialog_settings.xml` 
* In the value with this key `XML_STATE_EASY_IMPORT`

Easy import is managed with `net.tourbook.importdata.EasyImportManager` which is also saving/loading the config

**Comment by FJBDev** [comment]
Actually, I found that changing the value of this variable activates/desactivates the selected profile

`isActiveConfig=&quot;true`

It's not easy to change it but it's my only solution as of today to make my different profiles work.

The person from SO opened a bug in SWT [here](https://bugs.eclipse.org/bugs/show_bug.cgi?id=573616)

**Comment by FJBDev** [comment]
> I just tested MT 22.1 with Ubuntu 20.04 and it works. I think that the change of the Webkit version in your Linux is causing this issue.

Which version of Webkit do you have on Ubuntu 20.04 ?

**Comment by wolfgang-ch** [comment]
How do I get the Webkit version, Ubuntu in not my home

**Comment by FJBDev** [comment]
Execute this command:

`apt-cache policy libwebkit2gtk-4.0-37`

**Comment by wolfgang-ch** [comment]
![image](https://user-images.githubusercontent.com/1283445/152385473-4d603004-dc8d-4275-9972-8486ce212b80.png)

**Comment by FJBDev** [comment]
This bug is marked as fixed in Eclipse 4.24 (2022-06) [here](https://bugs.eclipse.org/bugs/show_bug.cgi?id=573616)

**Comment by FJBDev** [comment]
I confirm that this issue is fixed in MT 22.6 (test version)
